src/battle_util.c

```diff
@@ -9396,7 +9396,7 @@ static inline u32 CalcMoveBasePowerAfterModifiers(u32 move, u32 battlerAtk, u32
         break;
     case HOLD_EFFECT_PUNCHING_GLOVE:
         if (gMovesInfo[move].punchingMove)
-           modifier = uq4_12_multiply(modifier, UQ_4_12(1.1));
+           modifier = uq4_12_multiply(modifier, 4506);
```

Comment:
What's this magic number about?

Reply:
It is the punching glove modifier.

Reply:
the hold effect param is a u8 and It is kinda not worth it to increase it just because of punching glove imo

Reply:
Fixed.

Solution provided by @mrgriffin